commit ed0795b43e3c58275408e11f77c66998dde3f32c
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Mon Apr 3 10:36:04 2017 +0800

    Don't fail the whole daemon when iw is not present
    
    Change-Id: Ie2178af6292edd27151e0f36dbc3412f93f360c4

diff --git a/miracast/iw.go b/miracast/iw.go
index 0617b916..dbe60c11 100644
--- a/miracast/iw.go
+++ b/miracast/iw.go
@@ -23,7 +23,7 @@ func ListWirelessInfo() (WirelessInfos, error) {
 	os.Setenv("PATH", "/sbin:"+envPath)
 	defer os.Setenv("PATH", envPath)
 	outputs, err := exec.Command("/bin/sh", "-c",
-		"exec iw list").CombinedOutput()
+		"if hash iw 2>/dev/null; then exec iw list; fi").CombinedOutput()
 	if err != nil {
 		return nil, err
 	}
